src/appkit.cs

```diff
@@ -10551,6 +10691,65 @@ interface NSImageView : NSAccessibilityImage, NSMenuItemValidation {
 		[Mac (11, 0)]
 		[Export ("symbolConfiguration", ArgumentSemantic.Copy)]
 		NSImageSymbolConfiguration SymbolConfiguration { get; set; }
+
+		// from NSSymbolEffect (NSImageView)
```

Comment:
Documentation says NSSymbolEffect is a base class, so why not bind it as such?

https://developer.apple.com/documentation/symbols/nssymboleffect

Reply:
Xtro reports the following: 

Looking at xtro we get:

!missing-selector! +NSImageView::defaultPreferredImageDynamicRange not bound
!missing-selector! +NSImageView::setDefaultPreferredImageDynamicRange: not bound
!missing-selector! NSImageView::addSymbolEffect: not bound
!missing-selector! NSImageView::addSymbolEffect:options: not bound
!missing-selector! NSImageView::addSymbolEffect:options:animated: not bound
!missing-selector! NSImageView::imageDynamicRange not bound
!missing-selector! NSImageView::preferredImageDynamicRange not bound
!missing-selector! NSImageView::removeAllSymbolEffects not bound
!missing-selector! NSImageView::removeAllSymbolEffectsWithOptions: not bound
!missing-selector! NSImageView::removeAllSymbolEffectsWithOptions:animated: not bound
!missing-selector! NSImageView::removeSymbolEffectOfType: not bound
!missing-selector! NSImageView::removeSymbolEffectOfType:options: not bound
!missing-selector! NSImageView::removeSymbolEffectOfType:options:animated: not bound
!missing-selector! NSImageView::setPreferredImageDynamicRange: not bound
!missing-selector! NSImageView::setSymbolImage:withContentTransition: not bound
!missing-selector! NSImageView::setSymbolImage:withContentTransition:options: not bound

We are looking at the category on the NSImageView class.